### DISCUSSION TITLE
Edit modal state without server-side state management

### DISCUSSION THREAD

**Discussion by maminskik** [livewire/livewire#8421] [1. Help]
Hi, Im developing an app using Livewire with Laravel Breeze and have implemented a modal provided by Breeze. 
It works well for adding a user, but I'm trying to figure out if it's possible to manage the state of an edit modal without server-side state management. 

I know that Jetstream uses a different approach where the modal is managed by server-side Livewire, and I've tried using it for edits, but it isn't as smooth as the one based on Alpine.js with dispatch, which takes over 300ms to respond.

 Is there a way to use the Breeze modal based on Alpine.js and dispatch, and only make a server request to fetch the user data when a user is clicked? Or would it be better to use Axios in this case? Any help would be appreciated.

**Comment by EdgarsJoja** [comment]
Hey, might be easier to help if some code is provided :). 
But generally, you can just use AlpineJS to manage FE state. As you said, it will be much more responsive vs creating req. to server for all the clicks, hovers and whatnot, that do not actually require any additional data from server.
```blade
<div x-data="{ show: false }">
    <button @click="show = !show">Toggle</button>
    <div x-show="show">Test</div>
<div>
```

I don't know laravel breeze modals, but if it has some JS API, then nothing stops you from calling it vis alpinejs (or plain JS) where necessary.
